features/main/uuid.feature

```diff
@@ -194,3 +194,12 @@ Feature: Using uuid identifier on resource
     Then the response status code should be 400
     And the response should be in JSON
     And the header "Content-Type" should be equal to "application/ld+json; charset=utf-8"
+
+  @!mongodb
```

Comment:
should we also test this with mongo ?

Reply:
No need to test it with mongo as symfony/uid uses doctrine/dbal which is only useful with doctrine/orm. Testing it with doctrine/odm will be irrelevant